hub/config/src/main/java/uk/gov/ida/hub/config/domain/IdentityProviderConfigEntityData.java

```diff
@@ -66,6 +65,11 @@ public String getEntityId() {
         return entityId;
     }
 
+    @Override
+    public String toString() {
+        return String.format("%s | %s", this.getEntityId(), this.getSimpleId());
```

Comment:
Why did you add this toString method?

Reply:
Yes, it was for the unit tests as otherwise the entities couldn't be identified in the error messages. Also helpful if we ever get an exception so the useful info would show up in the message instead of the random numeric id generated by JVM